### PR TITLE
chore!: remove noise pipes option

### PR DESCRIPTION
**Pull request by dapplion**
As discussed offline, not need to keep code that will probably be never used

BREAKING CHANGE: noise pipes option no longer supported

### CODE REVIEW

**Comment by mpetrunic** [comment]
@dapplion lint errors^^

**Comment by dapplion** [comment]
> @dapplion lint errors^^

So I had to disable prettier for this repo because it completely changed everything. But then there's this linter? So in Lodestar prettier is connected to the linter such that VSCode can understand what the linter wants and apply that automatically for you if you have  `{ "editor.formatOnSave": false }`. How do you handle that with aegir?

**Comment by mpetrunic** [comment]
> > @dapplion lint errors^^
> 
> So I had to disable prettier for this repo because it completely changed everything. But then there's this linter? So in Lodestar prettier is connected to the linter such that VSCode can understand what the linter wants and apply that automatically for you if you have `{ "editor.formatOnSave": true }`. How do you handle that with aegir?

For me it detected. Do you have this checked: 
image

**Comment by mpetrunic** [comment]
@dapplion since this is breaking change, should we wait with releasing until we land changed connection encrypted interface?

**Comment by mpetrunic** [comment]
> @dapplion since this is breaking change, should we wait with releasing until we land changed connection encrypted interface?

@wemeetagain^^

**Comment by wemeetagain** [comment]
yea lets wait, should happen soon

File: src/noise.ts [hunk 1]
```diff
@@ -220,7 +145,7 @@ export class Noise implements INoiseConnection {
       await handshake.exchange()
       await handshake.finish()
 
-      if (this.useNoisePipes && handshake.remotePeer) {
+      if (handshake.remotePeer) {
```
Comment (mpetrunic) [inline src/noise.ts:148]: this condition and keycache store should be removed

**Review by mpetrunic — CHANGES_REQUESTED** [review]

**Comment by dapplion** [comment]
> > > @dapplion lint errors^^
> > 
> > 
> > So I had to disable prettier for this repo because it completely changed everything. But then there's this linter? So in Lodestar prettier is connected to the linter such that VSCode can understand what the linter wants and apply that automatically for you if you have `{ "editor.formatOnSave": true }`. How do you handle that with aegir?
> 
> For me it detected. Do you have this checked: image

Still not working, I can DM to debug

**Comment by mpetrunic** [comment]
@achingbrain aegir dep-check is passing without warnings or errors for unused prod dependencies. Is that normal?

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**